Jenkinsfile

```diff
@@ -43,11 +43,13 @@ pipeline {
     stages {
         stage('Clone project') {
             steps {
+                sh 'rm -fr project'
```

Comment:
It's a bit uncommon to use the options in this order, why not "rm -rf project"?

Reply:
IMO it reads better, as you can think of it as "force the recursive removal". Plus, if in doubt, I tend to sort options alphabetically.

Reply:
But in the end I really do not care enough to make this a blocker for merging, so I'll change it if you want.